packages/clay-label/src/ClayLabel.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
Remove use strict

Reply:
Hey @carloslancha,

Why remove `use strict`?

Reply:
@matuzalemsteles in ES6 `use strict` is no needed, and because we're transpiling through Babel is already being added in the process

Reply:
Note, I see `use strict` in other ES6 projects. Should those be removed as well?